### ISSUE TITLE
License

### ISSUE THREAD

**Issue by francisrupert** [USDA-FSA/fsa-design-system#43]
### Task

Create this repo's **License.md**. 
### Details

Everything will in the **Public Domain** or **Free**, i.e. [Federal Source Code Policy](https://sourcecode.cio.gov/)

Most likely a straight copy of [18F/open-source-policy](https://github.com/18F/open-source-policy/blob/master/LICENSE.md) and/or heavily influenced by [18F/web-design-standards/LICENSE.md](https://github.com/18F/web-design-standards/blob/18f-pages-staging/LICENSE.md)
### Progress
- [x] Drafting
- [ ] Drafted
- [ ] Reviewed

**Comment by brianmillsUX** [comment]
added draft (but edited!) page in "getting started" section

**Comment by francisrupert** [comment]
Moved to its own branch `feature/license-and-contributing`. Duplicate issue (intentionally) of USDA-FSA/fsa-style#11

**Comment by francisrupert** [comment]
Intentionally a duplicate of https://github.com/USDA-FSA/fsa-style/issues/11 and therefore blocked by it.
